api-scanner/src/test/resources/kotlin-exclude-method/kotlin/net/corda/example/ClassWithExtraConstructorGenerated.kt

```diff
@@ -0,0 +1,10 @@
+@file:Suppress("UNUSED")
+package net.corda.example
+
+class ClassWithExtraConstructorGenerated private constructor(val val1: String, val val2: String) {
```

Comment:
Declaring `val1` and `val2` as `private` would remove them from the API output and so reduce the size of the test case.